presto-parser/src/main/java/com/facebook/presto/sql/SqlFormatter.java

```diff
@@ -912,6 +905,11 @@ private String formatSqlParameterDeclarations(List<SqlParameterDeclaration> para
                     .collect(joining(",\n", "(\n", "\n)"));
         }
 
+        public static String formatTypeList(List<String> types)
+        {
+            return format("(%s)", Joiner.on(",").join(types));
```

Comment:
I don't remember whether we discussed this or not. Is the convention with space after `,` or not?

Reply:
I did not discuss about this, but the argument list printed by `SHOW FUNCTION` is delimited by comma and space (`, `), I'll do the same here for consistency.